### PR TITLE
Align Confluent Platform version to 6.0.0

### PR DESCRIPTION
**Pull request by ennru**
Follow-up to #1106

### CODE REVIEW

**Comment by ennru** [comment]
References #1223

File: testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java [hunk 1]
```diff
@@ -32,6 +31,7 @@
 
   private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
 
+  // Align this with testkit/src/main/resources/reference.conf
```
Comment (ignasi35) [inline testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java:34]: Super off-topic: it'd be great if we had a comment indicating `CP` stands for `Confluent Platform`

**Review by ignasi35 — APPROVED** [review]

File: testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java (hunk 1 above)
Comment (ignasi35) [reply testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java:34]: I'm talking about the name of this constant: `DEFAULT_CP_PLATFORM_VERSION`

Comment (seglo) [reply testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java:34]: That's an artifact of this type which was from https://github.com/testcontainers/testcontainers-java and modified for our purposes.

**Pull request merged**